src/messages.js

```diff
@@ -38,6 +39,11 @@ exports.sendMessage = async (pool, userId, text, groups) => {
   }
   let binding = [];
 
+  const userPhoneNumber = (await usersApi.getUserIdPhone(pool, userId)).data[0]
+    .phone_number;
```

Comment:
If you always want the phone number you could just have this return a phone number instead of having the data bits etc have the function do it.